src/internet_identity/src/main.rs

```diff
@@ -631,6 +631,33 @@ mod v2_api {
         Ok(())
     }
 
+    #[update]
+    #[candid_method]
+    fn authn_method_metadata_replace(
+        identity_number: IdentityNumber,
+        public_key: PublicKey,
```

Comment:
maybe rename `public_key` to `authn_method_pk` or something similar, so that is clear what is the meaning of the provided public key?